packages/babel-plugin-transform-react-jsx-development/test/fixtures/cross-platform/generated-jsx/input.js

```diff
@@ -0,0 +1 @@
+// empty
```

Comment:
I have also moved the test from `"linux"` to `"cross-platform"` so that it is also tested on Windows. It should be os-agnostic because it does not involve `path` related features.